packages/browser-wallet/src/shared/utils/payload-helpers.ts

```diff
@@ -116,6 +119,11 @@ export function parsePayload(
                 },
             };
         }
+        case AccountTransactionType.Transfer:
+            return {
+                type,
+                payload: payload as SimpleTransferPayload,
+            };
```

Comment:
Why not just the default case?

Reply:
The type checker couldn't figure out that it works without this, because `Transfer` is a special case in the `HeadlessTransaction` type I guess.